src/ICal/ICal.php

```diff
@@ -688,7 +688,11 @@ public function iCalDateWithTimeZone(array $event, $key, $format = self::DATE_TI
 
         // Force time zone
         if (isset($dateArray[0]['TZID'])) {
-            $dateTime->setTimezone(new \DateTimeZone($dateArray[0]['TZID']));
+            if($this->isValidTimeZoneId($dateArray[0]['TZID'])){
+                $dateTime->setTimezone(new \DateTimeZone($dateArray[0]['TZID']));
+            }else{
+                $dateTime->setTimezone($this->defaultTimeZone);
+            }
```

Comment:
Next time please abide by the [PSR-2 standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md) already used by this repo